op-challenger/game/fault/trace/outputs/provider_test.go

```diff
@@ -0,0 +1,116 @@
+package outputs
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-node/testlog"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	prestateBlock       = uint64(100)
+	poststateBlock      = uint64(200)
+	prestateOutputRoot  = common.HexToHash("0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
+	firstOutputRoot     = common.HexToHash("0xbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb")
+	poststateOutputRoot = common.HexToHash("0xcccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc")
+)
+
+func TestGet(t *testing.T) {
+	t.Run("FirstBlockAfterPrestate", func(t *testing.T) {
```

Comment:
Probably should test requests for before the prestate block too. Probably just returns an error.

Reply:
Added a test for that by "misconfiguring" the prestate block number to `0`. Otherwise no way to query before the prestate since the parameter is incremented by the prestate block number plus 1.

Reply:
Oh that's my bad - I totally missed that point.  You don't really need that test then, sorry.